src/Unity/Helper/Unity.Data.cs

```diff
@@ -10,23 +13,80 @@ public partial class Unity
     public Module UnityPlayer { get; private set; }
     public SceneManager Scenes { get; private set; }
 
+    private string _dataFolder;
+    public string DataFolder
+    {
+        get
+        {
+            string directory = Path.GetDirectoryName(MainModule.FilePath);
+            string folderName = _dataFolder ?? Path.GetFileName(MainModule.FilePath)[..^4];
+
+            return Path.Combine(directory, folderName);
+        }
+        set
+        {
+            Validation.AssertAction(nameof(DataFolder), "startup");
+
+            Debug.Info($"  => Will use {value} as the DataFolder.");
+            _dataFolder = value;
+        }
+    }
+
     private bool _loadSceneManager;
     public bool LoadSceneManager
     {
         get => _loadSceneManager;
         set
         {
-            if (Actions.Current != "startup")
-            {
-                string msg = $"{nameof(LoadSceneManager)} may only be set in the 'startup {{}}' action.";
-                throw new InvalidOperationException(msg);
-            }
+            Validation.AssertAction(nameof(LoadSceneManager), "startup");
 
             Debug.Info(value ? "  => Will try to load SceneManager." : "  => Will not load SceneManager.");
             _loadSceneManager = value;
         }
     }
 
+    private List<string> _monoV1Modules = ["mono.dll"];
+    public List<string> MonoV1Modules
+    {
+        get => _monoV1Modules;
+        set
+        {
+            Validation.AssertAction(nameof(MonoV1Modules), "startup");
+            Validation.AssertNotNull(nameof(MonoV1Modules), value);
+
+            Debug.Info($"  => Will use the Mono V1 modules {string.Join(", ", value)}. ");
```

Comment:
this isn't really necessary. the user sets these manually, they already know what's going to be used.  
yes, i have prints like this elsewhere, and they should also be reduced.